infrastructure/zk/src/run/run.ts

```diff
@@ -10,16 +11,65 @@ import * as docker from '../docker';
+};
+
+type PartialFeeTickerConfig = {
+    fee_ticker: {
+        unconditionally_valid_tokens: string | string[];
+    };
+};
+
 export async function deployERC20(command: 'dev' | 'new', name?: string, symbol?: string, decimals?: string) {
     if (command == 'dev') {
-        await utils.spawn(`yarn --silent --cwd contracts deploy-erc20 add-multi '
-            [
-                { "name": "DAI",  "symbol": "DAI",  "decimals": 18 },
-                { "name": "wBTC", "symbol": "wBTC", "decimals":  8, "implementation": "RevertTransferERC20" },
-                { "name": "BAT",  "symbol": "BAT",  "decimals": 18 },
-                { "name": "GNT",  "symbol": "GNT",  "decimals": 18 },
-                { "name": "MLTT", "symbol": "MLTT", "decimals": 18 }
-            ]' > ./etc/tokens/localhost.json`);
+        await utils.spawn(`yarn --silent --cwd contracts deploy-erc20 add-multi '[
+            {
+              "decimals": 18,
+              "symbol": "RIF",
+              "name": "RSK Infrastructure Framework"
+            },
+            {
+              "decimals": 18,
+              "symbol": "rDOC",
+              "name": "rDOC"
+            },
+            { "name": "DAI",  "symbol": "DAI",  "decimals": 18 },
+            { "name": "wBTC", "symbol": "wBTC", "decimals":  8, "implementation": "RevertTransferERC20" },
+            { "name": "BAT",  "symbol": "BAT",  "decimals": 18 },
+            { "name": "GNT",  "symbol": "GNT",  "decimals": 18 },
+            { "name": "MLTT", "symbol": "MLTT", "decimals": 18 }
+          ]' > ./etc/tokens/localhost.json`);
+
+        // Update the fee ticker configuration file with the deployed unconditionally valid erc20 tokens
+        const localhostTokensFile = fs.readFileSync('./etc/tokens/localhost.json', 'utf-8');
+        const tokens: Token[] = localhostTokensFile && JSON.parse(localhostTokensFile);
+        const deployedUnconditionallyValidAddresses =
+            (tokens &&
+                tokens.filter((token) => ['rDOC', 'RIF'].includes(token.symbol)).map(({ address }) => address)) ||
+            [];
+
+        const feeTickerConfigFile = fs.readFileSync('./etc/env/dev/fee_ticker.toml', 'utf-8');
+        const feeTicker = feeTickerConfigFile && (toml.parse(feeTickerConfigFile) as PartialFeeTickerConfig);
+        let unconditionallyValidTokens = feeTicker && feeTicker.fee_ticker.unconditionally_valid_tokens;
+
+        if (!Array.isArray(unconditionallyValidTokens)) {
+            unconditionallyValidTokens = [unconditionallyValidTokens];
+        }
+        if (feeTicker) {
+            feeTicker.fee_ticker.unconditionally_valid_tokens = [
+                ...unconditionallyValidTokens,
+                ...deployedUnconditionallyValidAddresses
+            ];
+            const newConfig = toml.stringify(feeTicker);
+            fs.writeFileSync('./etc/env/dev/fee_ticker.toml', newConfig, 'utf-8');
+            console.log('updated fee ticker config with', newConfig);
```

Comment:
@jurajpiar I guess I'm missing something here, but it's not clear to me why we need to generate the `fee_ticker.toml` file dynamically, instead of configuring it directly using the base folder

Reply:
I think I am the one missing something here. what do you mean by
> ... configuring it directly using the base folder

?

Reply:
The token addresses are automatically generated so, we couldn't set it in the base folder. BTW after talking, we realised that we could use `sed` to change the file.

Reply:
I did try that, but realised that spawning a process just to write to file that we have already loadded into js seems ugly, and it would not even reduce code size. The line would look something like:
```suggestion
            await utils.spawn(`sed -i -E 's/^\(unconditionally_valid_tokens\s*=\s*\).*/\${1}$${newTokens}/' etc/env/dev/fee_ticker.toml`);
            console.log('updated fee ticker config with', newConfig);
```

Reply:
The only benefit of this approach would be that we'd not lose the comments in that file (as the toml-read object doesn't contain them). However, since this file is generated, that might not be a great issue (the comments are still present in the base file).